layout_catalog/src/index.html

```diff
@@ -0,0 +1,413 @@
+<!doctype html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product catalog</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
+    <header class="page__header">
+      <a href="#home" class="page__header_logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__list">
+            <a href="#Apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Smartphones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#Laptops & Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__catalog">
+      <div class="page__card page__card--1" data-qa="card">
```

Comment:
```suggestion
      <div class="card" data-qa="card">
```
make it a separate BEM block and don`t  use modifiers on card